src/Filter.php

```diff
@@ -46,7 +46,9 @@ public function addDirectoryToWhitelist($directory, $suffix = '.php', $prefix =
      */
     public function addFileToWhitelist($filename)
     {
-        $this->whitelistedFiles[realpath($filename)] = true;
+        if (file_exists($filename)) {
```

Comment:
This `file_exists()` check is intentionally missing as there are situations where a file that needs to be on the whitelist does not exist at the time the whitelist is created.